Formula/tsduck.rb

```diff
@@ -0,0 +1,45 @@
+class Tsduck < Formula
+  desc "MPEG Transport Stream Toolkit"
+  homepage "https://tsduck.io/"
+  url "https://github.com/tsduck/tsduck/archive/v3.28-2551.tar.gz"
+  sha256 "ff39c9b97e17a5ae06556a119e24783427a86b1bb5ac75c7ba0340c7bd07d02d"
+  license "BSD-2-Clause"
+  head "https://github.com/tsduck/tsduck.git", branch: "master"
+
+  depends_on "dos2unix" => :build
+  depends_on "gnu-sed" => :build
+  depends_on "grep" => :build
```

Comment:
Do you `Makefile`s call `sed` and `grep` as `gsed` and `ggrep`? If not, these dependencies do nothing.

Reply:
Yes, gsed and ggrep are used as such in makefiles, the BSD versions from macOS don't support the same options and do not have the same behaviour (eg. "sed -i")